src/renderer/ssf-api.ts

```diff
@@ -512,12 +512,9 @@ export class SSFApi {
   }
 
   /**
-   * Shows a banner that informs user that the screen is being shared.
+   * Shows a banner that informs the user that the screen is being shared.
```

Comment:
This method is now pretty much identical to `openScreenSharingIndicator`. Should we remove that one? Also I see `handleMessage` in `app-bridge.ts` says it's only for 1.5. Should that entire section of code perhaps be deleted? I suspect it's being used right now by the screen sharing but if we are making these changes I don't think anyone uses it then? @KiranNiranjan

Reply:
Only RTC and client 1.5 was using the app-bridge.ts, now I think we will need remove the complete code